packages/suite/src/components/wallet/TransactionItem/components/TargetAddressLabel.tsx

```diff
@@ -37,8 +38,13 @@ export const TargetAddressLabel = ({ target, type, accountMetadata }: TargetAddr
                     // eslint-disable-next-line react/no-array-index-key
                     <AddressLabeling key={i} address={a} />
                 ) : (
-                    // eslint-disable-next-line react/no-array-index-key
-                    <span key={i}>{accountMetadata?.addressLabels[a] || a}</span>
+                    <IOAddress
+                        // eslint-disable-next-line react/no-array-index-key
+                        key={i}
+                        txAddress={accountMetadata?.addressLabels[a] || a}
+                        showCopyIcon={false}
```

Comment:
I am not able to copy the address now

Reply:
927c969d518312c98e78b97f8db3b502ffaa6d23